src/Microsoft.Performance.SDK/Processing/TableConfiguration.cs

```diff
@@ -184,11 +184,15 @@ public IEnumerable<ColumnConfiguration> Columns
         ///     Gets the mapping of column roles to column guids, if any.
         ///     If there are no column roles, then this dictionary will
         ///     be empty.
+        ///     Only the base projection of a column can be assigned a role; variants
+        ///     of a column cannot be assigned a role.
```

Comment:
Just thinking about docs, this is a good reason to create a new column rather than a variant.